charts/datadog/templates/_container-agent.yaml

```diff
@@ -58,6 +58,7 @@
   env:
     {{- include "containers-common-env" . | nindent 4 }}
     {{- include "fips-envvar" . | nindent 4 }}
+    {{- include "processes-common-envs" . | nindent 4 }}
```

Comment:
Any reason not to guard this with a check of `runInCoreAgent`?

Reply:
Added it

Reply:
sorry i think we actually want to keep this as it was, for the effort to have all container configs replicated in the core agent

Reply:
No worries, I removed the guard.